Ochowo/iReporter
# (chore):write-test-for-API-endpoints[finishes #162493296]

**What does this PR do?**
Creates different test cases for the endpoints

**Description of the task to be completed**
It should test all endpoints on the app

**How should this be tested manually**
After you clone the repo, cd into it and run` npm install` to install node modules. Make sure you have `nodejs` and `express` installed. Afterwards, run `npm test` to run the test. 

**What are the relevant pivotal stories**
[#162472441](https://www.pivotaltracker.com/story/show/162493296)
